### PR TITLE
adding gradients to all sides

### PR DESCRIPTION
**Pull request by prateekbh**
Adds gradient to all sides of page as mentioned in #11848 last point

cc: @alanorozco @newmuis @choumx

### CODE REVIEW

File: extensions/amp-story/0.1/amp-story-desktop.css [hunk 1]
```diff
@@ -81,6 +81,34 @@ amp-story-page.i-amphtml-story-page-shown {
...
+  position: absolute;
+  width: 25vw;
+  top: 0;
+  height: 100vh;
+}
+
+[desktop] .next-container {
+  right: 0;
+  background: rgba(33,33,33,0)!important;
+  background: -moz-linear-gradient(left, rgba(33,33,33,0) 0%, rgba(33,33,33,0.98) 100%)!important;
```
Comment (alanorozco) [inline extensions/amp-story/0.1/amp-story-desktop.css]: Remove vendor prefixes.

File: extensions/amp-story/0.1/amp-story.js [hunk 2]
```diff
@@ -314,16 +314,23 @@ export class AmpStory extends AMP.BaseElement {
   buildButtons_() {
     const doc = this.element.ownerDocument;
     const nextButton = doc.createElement('button');
+    const nextButtonContainer = doc.createElement('div');
```
Comment (alanorozco) [inline extensions/amp-story/0.1/amp-story.js]: nit: this DOM tree is getting too complicated for a series of `createElement`'s. Consider using utilities found in `./simple-template` (see `system-layer.js` for an example).

File: extensions/amp-story/0.1/amp-story-desktop.css (hunk 1 above)
Comment (alanorozco) [inline extensions/amp-story/0.1/amp-story-desktop.css]: Remove vendor prefixes.

File: extensions/amp-story/0.1/amp-story.js (hunk 2 above)
Comment (prateekbh) [reply extensions/amp-story/0.1/amp-story.js]: Done 👍

File: extensions/amp-story/0.1/amp-story-desktop.css (hunk 1 above)
Comment (prateekbh) [reply extensions/amp-story/0.1/amp-story-desktop.css]: Done

Comment (prateekbh) [reply extensions/amp-story/0.1/amp-story-desktop.css]: Done

**Comment by prateekbh** [comment]
Actually this change looks fine with a background but not white. The gradient clearly appears choppy.

Any suggestions?

screen shot 2017-11-10 at 3 38 53 am

**Comment by prateekbh** [comment]
I discussed with Hong and and corrected the gradient values.. all fine now ✌️

File: extensions/amp-story/0.1/amp-story.js [hunk 3]
```diff
@@ -312,21 +344,16 @@ export class AmpStory extends AMP.BaseElement {
...
-
-    this.element.insertBefore(previousButton, this.element.firstChild);
-    this.element.insertBefore(nextButton, this.element.firstChild);
-
-    this.nextButton_ = nextButton;
-    this.prevButton_ = previousButton;
+    this.element.insertBefore(
+        renderSimpleTemplate(this.win.document, PAGE_SWITCH_BUTTONS),
+        this.element.firstChild
+    );
```
Comment (alanorozco) [inline extensions/amp-story/0.1/amp-story.js]: nit: `);` on previous line

Comment (prateekbh) [reply extensions/amp-story/0.1/amp-story.js]: Done

File: examples/amp-story/desktop-backgrounds.html [hunk 4]
```diff
@@ -54,6 +54,13 @@
           <p>This is the second page of this story.</p>
         </amp-story-grid-layer>
       </amp-story-page>
+
+      <amp-story-page id="page-3">
+          <amp-story-grid-layer template="vertical">
```
[the remark this answers is not in the document]
Comment (prateekbh) [reply examples/amp-story/desktop-backgrounds.html]: Done

**Pull request merged**